spec/ibge/reader/state_spec.rb

```diff
@@ -3,7 +3,24 @@
 describe Ibge::Reader::State do
   subject { described_class }
 
+  let(:columns) do
+    {
+      dtb: {
+        "0": :code,
+        "1": :state_name
+      },
+
+      sidra: {
+        "2": :state_federative_unit
+      }
+    }
+  end
+
   it 'should have defined columns' do
-    expect(subject.defined_columns).to include "0": :code, "1": :state_name
+    expect(subject.defined_columns).to eq columns
+  end
+
+  it 'should read from multiple sources' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.